io.github.am2rcommunitydevelopers.AM2RLauncher.appdata.xml

```diff
@@ -0,0 +1,35 @@
+<?xml version="1.0" encoding="UTF-8"?>
+<!-- Copyright 2022 AM2R Community Developers -->
+<component type="desktop-application">
+    <id>io.github.am2rcommunitydevelopers.AM2RLauncher</id>
```

Comment:
Given that it is `https://github.com/AM2R-Community-Developers` it should be `io.github.am2r_community_developers.AM2RLauncher`